bash/code_to_gast/bash_main.js

```diff
@@ -0,0 +1,12 @@
+const bash_router = require("./bash_router.js")
+const parse = require("bash-parser");
+
+const bash_input = process.argv[2]
+let input_ast = ""
+try {
+    input_ast = parse(bash_input);
+} catch(err) {
+    return "Error: code could not compile"
+}
+
+console.log(JSON.stringify(bash_router.node_to_gast(input_ast)))
```

Comment:
remove? JSON.stringify should probably not exist within the console.log since it is an important call.